guides/Getting Started/API/index.md

```diff
@@ -74,21 +72,140 @@ Example using openssl (note that the cert is a Base64 string and the ```CERTIFIC
 echo "MIIKMQIBAzCCCf..." | base64 -D > app-cert.p12 && openssl pkcs12 -nokeys -in app-cert.p12 -passin pass:CERTIFICATE_PASSPHRASE
 ```
 
+---
+### `POST /applications/certificate`
+This route updates the pkcs12 certificate of an application in the database.
+
+---
+### `GET /applications/groups`
+Returns the functional groups for which a given application has access.
+
+---
+### `PUT /applications/groups`
+Updates the functional groups for which a given application has access.
+
+---
+### `GET /applications/store`
+Retrieves approved, embedded application information, filterable by `uuid` or by `transport_type`. The possible values for `transport_type` are `webengine` and `websocket`. The return object includes app bundle information such as the location of the bundle and its file size, compressed and uncompressed. The logic of where to store these app packages is customizable by the policy server. See the `customizable/webengine-bundle/index.js` file for details.
+
+---
+### `POST /webhook`
+This is the route that should be specified on a company's page on the SDL Developer Portal (in the box titled Webhook URL under Company Info) to be hit by the SHAID server when an app has been updated.
+
+---
+### `POST /staging/policy` & `POST /production/policy`
+These are the routes sdl_core's default Policy Table should use when requesting a Policy Table update with either `/staging` or `/production` specified.
+Given a "shortened" Policy Table, the Policy Server will use that information to automatically construct a full Policy Table response and return it to the requester.
 
-## UI endpoints
+---
+### `GET /policy/preview`
+This is the route hit by the Policy Server UI requesting a preview of the Policy Table. A variable **environment** indicates whether it is to be staging or production.
+
+---
+### `POST /policy/apps`
+The Policy Server UI makes a request to this route which returns an example Policy Table segment for a particular app.
+
+---
+### `POST /permissions/update`
+This route queries SHAID for an update to the list of app permissions in the Policy Server.
```

Comment:
The route updates the available permissions and permission relationships from SHAID, and not app permissions.